loops_lab.js

```diff
@@ -0,0 +1,304 @@
+
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs
```

Comment:
Good demonstration of for and while loops and their implementations. Other than minor feedback with curly brackets, no major issues. Keep up the good work!